README.md

```diff
@@ -23,21 +21,72 @@ Please use [Package Control][pc] to install the linter plugin. This will ensure
 
 To install via Package Control, do the following:
 
-1. Within Sublime Text, bring up the [Command Palette][cmd] and type `install`. Among the commands you should see `Package Control: Install Package`. If that command is not highlighted, use the keyboard or mouse to select it. There will be a pause of a few seconds while Package Control fetches the list of available plugins.
+Within Sublime Text, bring up the [Command Palette][cmd] and type `install`. Among the commands you should see `Package Control: Install Package`. If that command is not highlighted, use the keyboard or mouse to select it. There will be a pause of a few seconds while Package Control fetches the list of available plugins.
+
+When the plugin list appears, type `codeclimate`. Among the entries you should see `SublimeLinter-contrib-codeclimate`. If that entry is not highlighted, use the keyboard or mouse to select it.
 
-1. When the plugin list appears, type `codeclimate`. Among the entries you should see `SublimeLinter-contrib-codeclimate`. If that entry is not highlighted, use the keyboard or mouse to select it.
+## How it works
+If the first opened folder in SublimeText contains a `.codeclimate.yml` configuration file in its root, `codeclimate` will recognize this file's settings.
+
+Suppose the `codeclimate` CLI finds no configuration file in the folder root: In that case, it will automatically run the default inspections of `structure` and `duplication`. If you have a `.codeclimate.yml` configuration file in a different folder, you can set SublimeLinter's `working_dir` setting (please see examples).
 
 ## Settings
 For general information on how SublimeLinter works with settings, please see [Settings][settings]. For information on generic linter settings, please see [Linter Settings][linter-settings].
 
+### Examples
+You can set the path to the `codeclimate` executable in the global SublimeLinter settings or your project settings:
+
+```json
+{
+  "linters": {
+    "codeclimate": {
+      "executable": "/usr/local/bin/codeclimate"
+    }
+  }
+}
+```
+
+If you want to ignore the configuration of a `.codeclimate.yml`,  for instance, to run a subset of codeclimate engines, you can set linter arguments in the global SublimeLinter settings or your project settings:
+
+```json
+{
+  "linters": {
+    "codeclimate": {
+      "args": [
+          "-e",
+          "structure",
+          "-e",
+          "duplication"
+        ]
+    }
+  }
+}
+```
+
+Suppose you use a `.codeclimate.yml`-configuration file. In that case, the `codeclimate` CLI needs to be executed in your configuration file's directory. Otherwise, it can't detect your configuration and runs only the default analyzes.
+
+SublimeLinter takes the (first) open folder (or the folder of a single opened file) as the working directory for executing its linter commands. You can change this behavior by setting the working directory of execution:
```

Comment:
To clarify:  It is possible to have multiple folders "open" in Sublime in one project/window.  SublimeLinter will then take the first matching folder, t.i. usually a folder the file belongs to.

Reply:
@kaste, thank you very much for your awesome support. I also updated the docs. Do you see further improvements in my PR?